src/MockHttpClient/HttpRequestMessageExtensions.cs

```diff
@@ -130,7 +132,104 @@ public static bool HasHeader(this HttpRequestMessage request, string headerName,
 
             return false;
         }
-    }
 
+        /// <summary>
+        /// Determines whether the request contains the specified content as string.
+        /// </summary>
+        /// <param name="request">the request</param>
+        /// <param name="expectedContent">the expected content</param>
+        /// <returns></returns>
+        public static bool HasStringContent(this HttpRequestMessage request, string expectedContent)
```

Comment:
This name makes it ambiguous as to whether its checking an exact match or just that this string exists in the content somewhere

Reply:
I tried to be consistent with the existing methods in this extension. What would be your proposal? 
`IsStringContent`?